openwebstart/src/main/java/com/openwebstart/os/mac/MacEntryFactory.java

```diff
@@ -66,19 +66,12 @@ public void createMenuEntry(final JNLPFile file) throws Exception {
             if(defaultIconLocations.isEmpty()) {
              final FavIcon favIcon = new FavIcon(file);
              return Optional.ofNullable(favIcon.download())
-                     .map(f -> f.getAbsolutePath())
-                     .map(p -> Collections.singletonList(p))
+                     .map(File::getAbsolutePath)
+                     .map(Collections::singletonList)
                      .orElse(Collections.emptyList()).toArray(new String[0]);
             }
             return defaultIconLocations.toArray(new String[0]);
         }
-        if(shortcutIconLocations.isEmpty()) {
```

Comment:
What happened to the loading of FavIcon ??

Reply:
what do you mean?

Reply:
The code is gone without replacement:

![gone](https://user-images.githubusercontent.com/1795224/69274895-9ee87800-0bdb-11ea-8928-4827d0c3cb68.png)